### PR TITLE
feat: support headerCollapsableOnly

### PR DESCRIPTION
**Pull request by Kermit-Xuan**
ref: https://github.com/react-component/collapse/pull/164

### CODE REVIEW

**Comment by Kermit-Xuan** [comment]
明天加上测试。

File: tests/index.spec.tsx [hunk 1]
```diff
@@ -486,4 +486,41 @@ describe('collapse', () => {
...
+      expect(collapse.find('.rc-collapse-header-text').exists()).toBeTruthy();
+      collapse.find('.rc-collapse-header').simulate('click');
+      expect(collapse.find('.rc-collapse-item-active').length).toBe(0);
+      collapse.find('.rc-collapse-header-text').simulate('click');
+      expect(collapse.find('.rc-collapse-item-active').length).toBe(1);
+    });
+    it('should read value of panel preferentially', () => {
+      const collapse = mount(
+        <Collapse headerCollapsableOnly>
+          <Panel headerCollapsableOnly={false} header="collapse 1" key="1">
```
Comment (afc163) [inline tests/index.spec.tsx:518]: Panel 上暂时不用加，目前看没发现需要单独配置的需求，以后有再加也行。

**Comment by afc163** [comment]
https://github.com/react-component/collapse/releases/tag/v2.1.0

**Comment by zombieJ** [comment]
ref https://github.com/ant-design/ant-design/pull/27496

**Pull request merged**